src/bin/tikv-ctl.rs

```diff
@@ -1119,6 +1168,40 @@ fn main() {
                         .help("column family names, combined from default/lock/write"),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("raw-scan")
+                .about("print all raw keys in the range")
+                .arg(
+                    Arg::with_name("from")
+                        .short("f")
+                        .long("from")
+                        .takes_value(true)
+                        .default_value("z")
+                        .help(raw_key_hint)
+                )
+                .arg(
+                    Arg::with_name("to")
+                        .short("t")
+                        .long("to")
+                        .takes_value(true)
+                        .default_value("\\173") // 'z' + 1
+                        .help(raw_key_hint)
+                )
+                .arg(
+                    Arg::with_name("limit")
+                        .long("limit")
+                        .takes_value(true)
+                        .default_value("30")
```

Comment:
This is kind of arbitrary, why did you choose `30`?

Reply:
I think by default it shouldn't print too much data to screen..

Reply:
OK. :)